### DISCUSSION TITLE
Is this startallback program safe?

### DISCUSSION THREAD

**Discussion by TonyHenrique** [microsoft/winget-pkgs#31163] [General]
I am having issues on Windows 11 because the 'Never' 'Combine taskbar buttons' setting disappeared in #Windows11

So I found this:
https://www.startisback.com/

Wich has this install method:

`winget install startallback`

Is it safe?

**Comment by vedantmgoyal9** [comment]
@TonyHenrique Windows Package Manager (aka WinGet) is a package manager by Microsoft just like Chocolatey, Scoop, etc. It just makes the installation of a program/application/software easy as you don't have to go to the software's website, click on the download button, run it and go through all the prompts.

If you are talking about the installation method (i.e. running `winget install startallback`), it is 100% safe because winget downloads the software from its "official source" and installs it on your computer.

Also, each software added by the community goes through several checks implemented by Microsoft to ensure that the software being added to the repository is completely safe and isn't a virus/malware. Only after all the checks are successfully passed, it is added. However, if you're still not satisfied whether the application is safe or a virus then you can google it and read its reviews, ratings and how many people have downloaded and used it.

In my opinion, StartIsBack/StartAllBack is safe since it has been used by a large number of people and has also been rated 5-star on Softpedia. You can visit [here](http://www.softpedia.com/reviews/windows/StartIsBack-Review-318442.shtml) to know more.

**Reply by jedieaston** [reply]
> has also been rated 5-star on Softpedia

Softpedia is full of malware. I wouldn’t trust their ratings if I were you. 

With that being said, startallback is fine.

**Reply by vedantmgoyal9** [reply]
Well, I heard that "_Softpedia is 100% safe until we download any software directly from their server_" (this isn't the case in WinGet as the downloads are directly from ISV's server) from my colleagues.

**Reply by username-is-not** [reply]
> 
> If you are talking about the installation method (i.e. running `winget install startallback`)

Yeah, is it not supposed to be: `winget install StartIsBack.StartAllBack --scope machine`

**Comment by asheroto** [comment]
I know this is old but wanted to chime in here.

StartAllBack is safe, I've used it for a long time without issue. I've contacted the developer with questions and they've been extremely responsive each time.
